compaction/compaction_manager.cc

```diff
@@ -710,6 +716,15 @@ void compaction_manager::register_metrics() {
                        sm::description("Holds the sum of compaction backlog for all tables in the system.")),
         sm::make_gauge("normalized_backlog", [this] { return _last_backlog / _available_memory; },
                        sm::description("Holds the sum of normalized compaction backlog for all tables in the system. Backlog is normalized by dividing backlog by shard's available memory.")),
+        sm::make_counter("validation_errors", [this] { return validation_errors_validate; },
```

Comment:
I know it's confusing, but all the description for a specific metric name should be the same